kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -8,7 +13,7 @@ server {
     gzip_proxied             expired no-cache no-store private auth;
     gzip_types               text/plain text/css text/javascript application/xml;
     client_max_body_size     {{ max_upload_size }};
-
+    include                  snippets/json-errors.conf;
```

Comment:
Call this `error-pages.conf`. These won't necessarily be just JSON errors. If the client requests HTML via the `Accept` header, we should return HTML.